packages/marble-api/openapis/marblecore-api.yaml

```
@@ -5339,6 +5397,48 @@ components:
           format: uuid
         status:
           $ref: '#/components/schemas/TestRunStatusDto'
+    TestRunDecisionData:
+      type: object
+      required:
+        - version
+        - outcome
+        - phantom_outcome
+        - score
+        - phantom_score
+        - total
+        - phantom_total
+      properties:
+        version:
+          type: string
+        outcome:
+          type: string
```

Je pense qu'il existe un type dans l'OAS pour outcome
```suggestion
        outcome:
          $ref: '#/components/schemas/OutcomeDto'
```